src/main/groovy/com/netgrif/application/engine/petrinet/domain/dataset/logic/action/ActionDelegate.groovy

```diff
@@ -244,7 +248,17 @@ class ActionDelegate {
                     this.outcomes.add(outcome)
                 }
             }]
-        }]
+        },
+         on: { Transition trans ->
```

Comment:
make method improvements are part of https://netgrif.atlassian.net/browse/NAE-1622 (currently in progress)